graylog2-server/src/main/java/org/graylog/plugins/views/search/views/ViewService.java

```diff
@@ -62,6 +63,11 @@ protected ViewService(MongoConnection mongoConnection,
         this.viewRequirementsFactory = viewRequirementsFactory;
         this.entityOwnerShipService = entityOwnerShipService;
         this.viewSummaryService = viewSummaryService;
+        for (String sortField : ViewDTO.SORT_FIELDS) {
```

Comment:
Would it make sense to run this code inside a migration, only once?

Reply:
Hmm, I feel it should be run on every start-up.
If the index is there, Mongo won't duplicate it, so there is nothing to worry about.
If it is not there (i.e. smb has dropped it), we recreate it.

Reply:
I checked again and we are doing the same in other services. Let's keep it as it is. Thanks!